File: C - language/Q13.c
#include <stdio.h>

int main() {
    int arr[]={1,2,3,4,5};
    int findnum=3;
    int len=sizeof(arr)/sizeof(arr[0]);
    int i,count;
    for (i=0;i<len;i++){
        if (arr[i]==findnum){
            count=count+1;
        }
    }
    if (count>0){
        printf("true");
    }else {
        printf("false");
    }

    return 0;
};

File: javascript/Q13.js
var arr=[1, 2, 3, 4, 5] 
var num=5;
var count=0;

for (i=0;i<arr.length;i++){
    if(arr[i]==num){
        count++
    }
}
if(count>0){
    console.log("true")
}else {
    console.log("false")
}